source/_docs/guides/github-pull-requests.md

```diff
@@ -0,0 +1,340 @@
+
+    ![Update configuration](/source/docs/assets/images/pr-workflow/update-configuration.png)
+
+    Visit your site's Pantheon dashboard, and go to the `pr-slogan` multidev page. Note that there are a handful of modified files here ready to be committed now. Type in a brief description of what you changed, and click **Commit**.
+
+Once the Pantheon dashboard finishes committing the code, visit your project page on GitHub. Go to your `slogan` pull request. Note that your commit has been added to this pull request, and the Circle CI status indicates that your tests are running. Whenever you commit files from the Pantheon dashboard, the commit will be reduced to contain only those files that belong in the GitHub repository, and this commit will be pushed back to the canonical repository. GitHub will then start a new Circle CI build, and the build results will once again be pushed to the existing multidev environment that was created for this branch. You may continue working in this environment, making multiple changes, and committing updates whenever you would like your tests to run again.
+
+## Update Your Project
+
+When using the Composer workflow, you should *never* use the Pantheon dashboard to update changes from your upstream, nor should you ever merge code from one environment to another. All code updates will be done using Composer. Composer commands (e.g. `composer update`) may be run directly against your Pantheon multidev environments using the Terminus Composer plugin.
+
+If you would like to copy the commands used in the examples below directly into your terminal, export environment variables to define your site name and multidev environment:
+```
+export SITE=my-pantheon-project
+export ENV=pr-slogan
+```
+1.  Using Terminus, update your site with Composer:
+
+    ```bash
+    terminus composer $SITE.$ENV -- update
+    ```
+
+    The example below shows a site that was installed with Drupal 8.3.0, and updated to Drupal 8.3.1 after that version was released using Composer.
+
+    ![Update configuration](/source/docs/assets/images/pr-workflow/composer-update.png)
+
+2.  Visit your Pantheon dashboard and commit your changes:
+
+    ![Commit updated files](/source/docs/assets/images/pr-workflow/commit-composer-update.png)
+
+## Add a New Module
+
+In this workflow, Composer should also always be used to install new modules and themes on your site. Never use the Drupal **Extend** -> **Install new module** feature, nor should you use `drush pm-download`, as neither of these techniques modify the `composer.json` file, which means that modules added this way will disappear the next time the build artifacts are pushed to your Pantheon multidev environment.
```

Comment:
This is my sentence. It is a little run-on-ish. Fixing not critical, and could happen as a follow-on change. I'll submit a suggestion if I have time.